build.gradle

```diff
@@ -15,3 +17,22 @@ task wrapper(type: Wrapper) {
 dependencies {
   testCompile 'junit:junit:4.12'
 }
+
+publishing {
+  repositories {
+    maven {
+      url "s3://casetext-ds-code-deploys/maven"
+      credentials(AwsCredentials) {
+        accessKey "$System.env.AWS_ACCESS_KEY_ID"
+        secretKey "$System.env.AWS_SECRET_ACCESS_KEY"
+      }
+    }
+  }
+
+  publications {
+    fnvProfileSignature(MavenPublication) {
+      artifactId 'fnv-profile-signature'
```

Comment:
Nope.  Caught this when I tried to build crick with the new repo.  Fixed.

On Tue, Jul 12, 2016 at 11:36 AM, Tom G Varik <notifications@github.com>
wrote:

> In build.gradle
> <https://github.com/casetext/fnv-text-profile-signature/pull/3#discussion_r70496076>
> :
>
> > @@ -15,3 +17,22 @@ task wrapper(type: Wrapper) {
> >  dependencies {
> >    testCompile 'junit:junit:4.12'
> >  }
> > +
> > +publishing {
> > +  repositories {
> > +    maven {
> > +      url "s3://casetext-ds-code-deploys/maven"
> > +      credentials(AwsCredentials) {
> > +        accessKey "$System.env.AWS_ACCESS_KEY_ID"
> > +        secretKey "$System.env.AWS_SECRET_ACCESS_KEY"
> > +      }
> > +    }
> > +  }
> > +
> > +  publications {
> > +    fnvProfileSignature(MavenPublication) {
> > +      artifactId 'fnv-profile-signature'
>
> The name of the published artifact has been fnv-text-profile-signature
> until now (e.g., crick#build.gradle:46
> <http:///casetext/crick/blob/master/build.gradle#L46>). Are you intending
> to change the name here?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/casetext/fnv-text-profile-signature/pull/3/files/20f3109267bd56bb764c4e2cdd85b2bd6e6d5ced#r70496076>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AFmhf0mm2uJuCT8o11icxk0ZSWGtMv5vks5qU96YgaJpZM4JKroL>
> .
>